res/openvpn-gui-res-it.rc

```diff
@@ -101,10 +102,24 @@ BEGIN
     LTEXT "", ID_TXT_IP, 20, 160, 300, 10
     PUSHBUTTON "&Disconnetti", ID_DISCONNECT, 50, 190, 50, 14
     PUSHBUTTON "&Riconnetti", ID_RESTART, 150, 190, 50, 14
-    PUSHBUTTON "De&tach", ID_DETACH, 180, 190, 50, 14, WS_DISABLED
+    PUSHBUTTON "&Stacca", ID_DETACH, 180, 190, 50, 14, WS_DISABLED
     PUSHBUTTON "&Nascondi", ID_HIDE, 100, 190, 50, 14
 END
 
+/* Message Window.
+ * This dialog needs no translation unless a different font instead
```

Comment:
Please do not add this dialog in this file as explained in the comment line 110. The one in the default (English) version is enough. 

In fact, if you start with your previous PR where these things were already settled, and add new stuff to it, we can avoid duplication of such comments and fixups.